db/schema.rb

```diff
@@ -665,6 +665,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "country_band_details", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.integer "number", null: false
+    t.date "start_date", null: false
+    t.date "end_date"
+    t.integer "due_amount_per_competitor_in_cents", null: false
```

Comment:
Maybe name this column `due_amount_per_competitor_us_cents` to make clear that the currency is pre-defined?

Reply:
Okay changed.